roles/gpu_operator_deploy_from_operatorhub/tasks/deploy_from_bundle.yml

```diff
@@ -15,18 +15,17 @@
       --ignore-not-found=true
 
 - name: Deploy the GPU Operator from the bundle
-  block:
-  - name: Deploy the GPU Operator from the bundle
-    command:
-      operator-sdk run bundle
-        -n {{ deploy_bundle_namespace }}
-        {{ deploy_bundle_image }}
-  rescue:
-  - name: Fix the GPU Operator image
-    command:
-      oc set image deployment/gpu-operator
-         gpu-operator={{ deploy_bundle_operator_image }}
-         -n {{ deploy_bundle_namespace }}
+  command:
+    operator-sdk run bundle
+                 -n {{ deploy_bundle_namespace }}
+                 {{ deploy_bundle_image }}
+
+- name: Fix the GPU Operator image
+  command:
+    oc set image deployment/gpu-operator
+       gpu-operator={{ psap_quay_master_operator }}
+       -n {{ deploy_bundle_namespace }}
+  when: deploy_bundle_namespace == psap_quay_master_bundle
```

Comment:
Can you explain this condition?

Why would `deploy_bundle_namespace` be equal to `psap_quay_master_bundle`

Reply:
wait, this hunk/commit is a) wrong b) not relevant anymore :|

correct condition was:
```
   when: deploy_bundle_image == psap_quay_master_bundle
```
but I couldn't get this to work properly, see https://github.com/openshift-psap/ci-artifacts/pull/203.

I'll replace this commit by the one in https://github.com/openshift-psap/ci-artifacts/pull/203 as it's still the closest to what I have in mind